### PR TITLE
UX changes to create intervention policy

### PR DESCRIPTION
**Pull request by asylves1**
# Description
 
image
image
image

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue [hunk 1]
```diff
@@ -426,6 +453,12 @@ const onSaveInterventionPolicy = async () => {
 	}
 };
 
+const createNewInterventionPolicy = () => {
+	if (!model.value?.id) return;
+	newBlankInterventionPolicy.value.modelId = model.value?.id;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue:458]: You just tested that it was not null or undefined
```suggestion
	newBlankInterventionPolicy.value.modelId = model.value.id;
```

File: packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue [hunk 2]
```diff
@@ -205,7 +223,14 @@ const knobs = ref<BasicKnobs>({
 	}
 });
 
+const newBlankInterventionPolicy = ref({
+	name: '',
+	modelId: '',
+	interventions: []
+});
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue:230]: Why not using the [blankIntervention](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/services/intervention-policy.ts#L5-L11)?

File: packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-card.vue [hunk 3]
```diff
@@ -289,6 +305,15 @@ const debounceUpdateState = debounce((intervention) => {
...
 }
 
+.type-menu {
+	border-radius: 5px 0 0 5px;
+	background: var(--surface-200);
+}
+
+.applied-to-menu {
+	border-radius: 0 5px 5px 0;
+}
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-card.vue:315]:
```suggestion
.type-menu {
	border-radius: var(--border-radius) 0 0 var(--border-radius);
	background: var(--surface-200);
}

.applied-to-menu {
	border-radius: 0 var(--border-radius) var(--border-radius) 0;
}
```

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue [hunk 4]
```diff
@@ -145,6 +155,14 @@
...
 		@on-save="onSaveAsInterventionPolicy"
 	/>
+	<tera-save-asset-modal
+		initial-name="New Intervention Policy"
+		:is-visible="showCreatePolicyModal"
+		:asset-type="AssetType.InterventionPolicy"
+		:asset="newBlankInterventionPolicy"
+		@close-modal="showCreatePolicyModal = false"
+		@on-save="onSaveAsInterventionPolicy"
+	/>
```
Comment (blanchco) [inline packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue:165]: can we use the modal that we currently have just above this block?

**Review by blanchco — APPROVED** [review]

**Pull request merged**